run.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+sudo -v
+if [ $? == 1 ]
+then
+    echo "[ERRO][TNGFUE] Without root permission, you cannot run TNGFUE"
+    exit 1
+fi
+
+PID_LIST=()
+IFACE_NAME='' # to store the network interface name used to connect to the TNGF / Wi-Fi AP
+IFACE_IP='192.168.1.202' # IP address to be used by the network interface
+IFACE_MASK='24' # network mask to be used by configuration command
+IFACE_BROADCAST_IP='192.168.1.255' # IP address to be used by configuration command
+ENABLE_DEBUG=0 # controls if debug messages are enabled
+
+case $1 in # insert a 'case statement' to allow adding other input parameters in the future
+    # TODO delete the comment above whenever a new parameter is added
+    -debug)
+    ENABLE_DEBUG=1
+    ;;
+esac
+
+function terminate()
+{
+    echo "[INFO][TNGFUE] Terminating TNGFUE..."
+    sudo kill -SIGTERM ${PID_LIST[@]}
+    sleep 3 # gives enough time for wpa_supplicant to finish when it's stuck or in a loop
+
+    # Remove all GRE interfaces
+    echo "[INFO][TNGFUE] Removing all GRE interfaces"
+    GREs=$(ip link show type gre | awk 'NR%2==1 {print $2}' | cut -d @ -f 1)
+    for GRE in ${GREs}; do
+        sudo ip link del ${GRE}
+        echo del ${GRE}
+    done
+
+    # Prepare XFRM to terminate
+    sudo ip xfrm policy flush
+    sudo ip xfrm state flush
+
+    # Remove all XFRM interfaces
+    echo "[INFO][TNGFUE] Removing all XFRM interfaces"
+    XFRMIs=$(ip link show type xfrm | awk 'NR%2==1 {print $2}' | cut -d @ -f 1)
+    for XFRMI in ${XFRMIs}; do
+        sudo ip link del ${XFRMI}
+        echo del ${XFRMI}
+    done
+
+    # Remove IP and route
+    echo "[INFO][TNGFUE] Removing IP and route"
+    sudo ip addr flush $IFACE_NAME # removes IP and default route too
+
+    cd ..
```

Comment:
```suggestion
    cd $TNGFUE_PATH
```

Reply:
And I think it's not necessary to cd back.

Reply:
If so, the `$TNGFUE_PATH` could be remove also.

Reply:
Hello, indeed it's not necessary to cd back.

When I added this cd command I was thinking of designing run.sh in a different way. However after implementing it with the current design this command was left behind and is unnecessary because as soon as the script ends, the shell is back on the path where the script was invoked.

Your suggestions were applied on https://github.com/free5gc/tngfue/pull/2/commits/443e4d123e4ba24fd72110e0743e445a541fb447

Thanks for the review